server/topic.go

```diff
@@ -306,24 +306,31 @@ func (t *Topic) runLocal(hub *Hub) {
 
 			// userId.IsZero() == true when the entire session is being dropped.
 			var asUid types.Uid
+			var isChan bool
```

Comment:
Maybe rename `isChanReq` in order to make a clear distinction from `t.isChan`?

Reply:
done `isChan` -> `asChan`